src/org/sosy_lab/java_smt/solvers/bitwuzla/BitwuzlaFormulaCreator.java

```diff
@@ -305,6 +313,10 @@ private FunctionDeclarationKind getDeclarationKind(Term term) {
       return FunctionDeclarationKind.FP_CASTTO_UBV;
     } else if (kind.equals(Kind.BV_XOR)) {
       return FunctionDeclarationKind.BV_XOR;
+    } else if (ImmutableList.of(Kind.BV_ROL, Kind.BV_ROLI, Kind.BV_ROR, Kind.BV_RORI)
+        .contains(kind)) {
+      // Rotations are not yet supported by FunctionDeclarationKind
+      return FunctionDeclarationKind.OTHER;
```

Comment:
Please do not just add such a mapping here, but report this as a bug for JavaSMT. The FunctionDeclarationKinds are missing on our interface.

Reply:
I've merged master with the new changes and removed this workaround